controllers/managedocs_controller_test.go

```diff
@@ -1594,6 +1610,15 @@ var _ = Describe("ManagedOCS controller", func() {
 					utils.WaitForResource(k8sClient, ctx, cephIngressNetworkPolicyTemplate.DeepCopy(), timeout, interval)
 				})
 			})
+			When("there is onboarding-validation-key field in the add-on parameters secret", func() {
+				It("should create onboarding-ticket-key secret", func() {
+					if testReconciler.DeploymentType != providerDeploymentType {
+						Skip(fmt.Sprintf("Skipping the test as it is not required by %v", testReconciler.DeploymentType))
+					}
+					secret := onboardingValidationKeySecretTemplate.DeepCopy()
+					Expect(k8sClient.Get(ctx, utils.GetResourceKey(secret), secret)).Should(Succeed())
+				})
+			})
```

Comment:
This test does not make sense? 
The secret is created whenever we deploy with the correct params, as such this needs to be checked together with the other resources that are created in that test.

Here you have no set up, so it cannot be a `when`